QRCoder/BitmapByteQRCode.cs

```diff
@@ -53,13 +53,35 @@ public byte[] GetGraphic(int pixelsPerModule, byte[] darkColorRgb, byte[] lightC
     {
         var sideLength = QrCodeData.ModuleMatrix.Count * pixelsPerModule;
 
-        var moduleDark = darkColorRgb.Reverse();
-        var moduleLight = lightColorRgb.Reverse();
+        var moduleDarkSingle = darkColorRgb.Reverse().ToArray();
+        var moduleLightSingle = lightColorRgb.Reverse().ToArray();
 
-        var bmp = new List<byte>();
+        // Pre-calculate color/module bytes
+        var moduleDark = new byte[pixelsPerModule * 3];
+        var moduleLight = new byte[pixelsPerModule * 3];
+        for (int i = 0; i < pixelsPerModule; i++)
+        {
+            Array.Copy(moduleDarkSingle, 0, moduleDark, i * 3, 3);
+            Array.Copy(moduleLightSingle, 0, moduleLight, i * 3, 3);
```

Comment:
eliminating `moduleDarkSingle` and changing these two lines to six lines would eliminate 2 or 3 allocations.  e.g. `moduleDark[i * 3] = darkColorRgb[2];` and so on.  probably improve performance too.